updateLocalesHelper/updateLocaleStatus.py

```diff
@@ -1,59 +1,92 @@
 #! /bin/env python
-import json
+from configparser import ConfigParser
+from http import HTTPStatus
+from json import dump
+from os import path, environ
+from requests import get as GET
+import sys
+
 from txclib import utils
 from txclib import config
-import requests
-import os
-import sys
-import configparser
 
-script_path = os.path.dirname(os.path.realpath(sys.argv[0]))
 
-status_file = os.path.join(script_path, '../src/locales/translationStatus.json')
-config_file = os.path.join(script_path, '../.tx/config')
+STATUS_FILE_PATH = '../src/locales/translationStatus.json'
+CONFIG_FILE_PATH = '../.tx/config'
+SCRIPT_PATH = path.dirname(path.realpath(sys.argv[0]))
+TRANSLATION_URL = 'https://api.transifex.com/organizations/yunity-1/projects/karrot/'
+DEFAULT_TOKEN_URL = 'https://www.transifex.com'
+
+STATUS_FILE = path.join(SCRIPT_PATH, STATUS_FILE_PATH)
+CONFIG_FILE = path.join(SCRIPT_PATH, CONFIG_FILE_PATH)
+
 
 def get_token():
-    token = os.environ.get('TX_TOKEN')
+    token = environ.get('TX_TOKEN')
     if token:
         return token
 
     txrc_file = utils.get_transifex_file()
     txrc = config.OrderedRawConfigParser()
     txrc.read((txrc_file,))
-    return txrc.get('https://www.transifex.com', 'token')
+    return txrc.get(DEFAULT_TOKEN_URL, 'token')
 
-token = get_token()
 
-r = requests.get('https://api.transifex.com/organizations/yunity-1/projects/karrot/',
-  auth=('api', token)
-)
-
-if r.status_code != 200:
-    print('Failed to get translation status!')
-    print('Response is:')
-    print(r.json())
-    sys.exit(1)
-
-stats = r.json()['stats']
-percentage_dict = {lang: round(v['translated']['percentage']*100) for lang, v in stats.items()}
-percentage_dict['en'] = 100
-
-config = configparser.ConfigParser()
-config.read(config_file)
-lang_map_raw = config.get('main', 'lang_map')
-lang_map = {}
-for arg in lang_map_raw.replace(' ', '').split(','):
-    k, v = arg.split(":")
-    lang_map.update({k: v})
-
-# translate to HTTP Accept-Language format
-def translate_key(key):
+def get_translation_status(token):
+    response = GET(TRANSLATION_URL, auth=('api', token))
+    if response.status_code != HTTPStatus.OK:
```

Comment:
As we are using `requests` here, might use their abstraction: https://requests.kennethreitz.org/en/master/user/quickstart/#response-status-codes

```suggestion
    if response.status_code != requests.codes.ok:
```